ws/javaee8/README.md

```diff
@@ -0,0 +1,55 @@
+## Java EE 8 Sample Applications README
+
+This README file contains step-by-step instructions which can be used to migrate existing Java EE 7 sample applications to Java EE 8 sample application workspace. Also describes how to add a new example.
```

Comment:
I think the instructions should be about adding (and not porting) new samples. As part of these changes, you have already ported all existing samples. So for an end user, its more useful to have instructions on how to add samples.